### PR TITLE
added sound

### PR DESCRIPTION
**Pull request by ghost**
I added some sound.. maybe you could add a toggle for sound on/sound off

### CODE REVIEW

**Comment by livibetter** [comment]
I have to decline this PR, because this PR is a quick hack and it doesn't probably handle the things below:

1. It should be an option, and check required dependency: `play`.

   The current code doesn't have option parsing in place, so another PR for options must be taken care of first.

1. The background `play` process must be properly terminated when the main `rain.sh` exits.

1. `play` may spill some messages, as it did on my system, "`play WARN alsa: can't encode 0-bit Unknown or not applicable`", this must be handled in some way, or the messages will be artifacts for a while.

In summary, a couple of things has to be done first:

1. Move the main part into `main` function and

1. Add option parsing functionality.

Then, finally this PR (or open new one) with a few fixes for those issues.

I am closing this, because I don't want to do those and don't expect you would be interested in submitting more PRs/commits, but if you are, that's brilliant.  However, please do first discuss with me in more detail on what exactly what needs to be done.

*FWIW, I quite like the soothing raining sound.*
